admin_manual/installation/source_installation.rst

```diff
@@ -354,15 +334,15 @@ Required Extensions
 
 ::
 
-  zypper install apache2 apache2-mod_php5 php5-gd php5-json php5-curl \
-    php5-intl php5-mcrypt php5-zip php5-zlib
+  zypper install -y apache2 apache2-mod_php7 php7-gd \
```

Comment:
please double check if there is a package called php7-openssl. we had a report about this with opensuse https://github.com/owncloud/core/issues/30337

Reply:
Shall do.